service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +40,57 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public AssessmentResult insertAssessmentResult(
+      UUID id, String assessmentResult, String veteranIcn, String diagnosticCode)
```

Comment:
assesmentResult has both veteranIcn and diagnosticCode fields so I am not sure why we need them in the signature. See the comment a few lines below.

Reply:
The response from the controller in the end does have the ICN and the diagnostic code, but this assessment result is just the evidence from Primary Routes.  I double checked to see what is in it when it is sent to SaveToDb and this is what we get:

This is the assessment result{"evidence": {"medications": [{"status": "Active", "notes": ["Naproxen sodium 220 MG Oral Tablet"],   ETC 

I will look into seeing if I can get the entire response.